### ISSUE TITLE
Feature Extraction from images

### ISSUE THREAD

**Issue by adithyakumarcr** [sagarVL/TUHH_ISM_tumor_detection#14]
- [ ] Color Histogram
 - [ ] Moments
 - [ ]  HuMoments
 - [ ]  Cooccurance matrices
 - [ ] HOG gradients.

**Comment by adithyakumarcr** [comment]
Finished with feature extraction, all though they are useless. The feature extraction was conducted on training dataset. No image processing was done in prior. Feature extraction with image processing will be my next week task. 

The CSV files have been updated onto branch feature extraction.
